config/lobby/db/004_audit_bans

```diff
@@ -0,0 +1,23 @@
+start transaction;
+
+alter table banned_macs
+  add column mod_username varchar(40),
+  add column mod_ip varchar(45),
+  add column mod_mac char(28),
+  add constraint banned_macs_mod_mac_check check (char_length(mod_mac)=28);
```

Comment:
The new columns allow `NULL` values to accommodate existing bans.  In the future, once all legacy bans have expired, we can revisit making these columns `NOT NULL`.  Alternatively, we could add bogus values for all existing records as part of the migration, but I'm not sure if that's appropriate.

Ditto for the `banned_usernames` table.

Reply:
I'd actually prefer the bogus values like "unkown" and 0.0.0.0 (or something like 127.0.0.1/ [::1]) and '0000000000000000000000000000' over just leaving everything as-is.
If just a single ban is "forever", these rows will never be changed to require non-null values (and I know there are some).

Reply:
Ok, I'll update the script to use dummy values.  Would it be preferable to

1. Add the columns without the `NOT NULL` constraint.
1. `UPDATE` existing records with the dummy values.
1. Alter the columns to add the `NOT NULL` constraint.

or

1. Add the columns with a `NOT NULL` constraint and a default value corresponding to the dummy values.
1. Alter the columns to remove the default value.

Reply:
I'd be fine with both attempts, but I'd prefer option 2.